src/frontend/screens/Settings/MapSettings/BackgroundMaps.tsx

```diff
@@ -138,6 +141,7 @@ export const BackgroundMaps: NativeNavigationComponent<"BackgroundMaps"> = ({
             isSelected={styleUrl === defaultStyleUrl}
             styleUrl={defaultStyleUrl}
             mapTitle="Default Map"
+            bytesStored={0}
```

Comment:
Is there a way to get the total bytesStored for the default map?

Reply:
not entirely sure but would probably be a hardcoded number. i _think_ that lives in https://github.com/digidem/mapeo-server but can't fully remember

Reply:
i think what you have here is fine though, i'm doing the same thing in https://github.com/digidem/mapeo-mobile/pull/1031 and just not displaying the info if the amount is 0. since you can't delete this map anyways, i think it's not as important to include the storage amount for it

Reply:
Right...Then i'll just not show it!